### PR TITLE
The first file to play with Github

### PR DESCRIPTION
**Pull request by wdzfortune**
desc...

### CODE REVIEW

File: wdzfortune.md [hunk 1]
```diff
@@ -7,3 +7,5 @@
 ## newly added line to compare difference
```
Comment (rs186002) [inline wdzfortune.md:8]: That's a pretty empty space

Comment (rs186002) [inline wdzfortune.md:8]: It has a deep meaning

**Comment by wdzfortune** [comment]
Deap meaning? @rs186002

**Comment by rs186002** [comment]
@wdzfortune you've been officially notified!

**Comment by rs186002** [comment]
@wdzfortune lets try again

**Comment by wdzfortune** [comment]
@rs186002 change is made per your request.

**Pull request merged**